### PR TITLE
docs: Add AWS region enablement documentation

### PR DESCRIPTION
**Pull request by aaradhak**
Added a document detailing the steps for enabling AWS regions. 
Updated the check-aws-regions.Jenkinsfile to include a link to the new documentation.

### CODE REVIEW

File: docs/aws-region-enable.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    To Enable a region:
+    Sign in to the AWS Management Console.
+    In the upper right corner of the console, choose your account name or number and then choose My Account.
+    In the AWS Regions section, next to the name of the Region that you want to enable, choose Enable.
+    In the dialog box, review the informational text and choose Enable Region.
+    Wait until the Region is ready to use.
+
+    We would also like to get AWS public access enabled for this region. Please use the following command:
+        "aws ec2 disable-image-block-public-access --region `<missing-region>`"
```
Comment (jbtrystram) [inline docs/aws-region-enable.md:31]: let's also ask for the qota increase so all the regions have the same settings : 

```suggestion
        "aws ec2 disable-image-block-public-access --region `<missing-region>`"
Let's also adjust the qota to match the other regions : 
        "aws service-quotas request-service-quota-increase --region=`<missing-region` --service-code=ec2 --quota-code=L-0E3CBAB9 --desired-value 1000.00"        
```

**Review by jbtrystram — CHANGES_REQUESTED** [review]
for completess let's ask in the same ticket to adjust the qota, as per https://pagure.io/fedora-infrastructure/issue/11707#comment-891844

**Review by jbtrystram — APPROVED** [review]
lgtm

File: jobs/check-aws-regions.Jenkinsfile [hunk 2]
```diff
@@ -1,3 +1,5 @@
+/* Note: AWS region enablement, refer to the documentation here: [AWS Region Enablement](docs/aws-region-enable.md) */
```
Comment (dustymabe) [inline jobs/check-aws-regions.Jenkinsfile:1]: I feel like maybe we should include the link (i.e. clickable) in the slack message that gets sent to the channel.

i.e. the notification sends us somewhere with the steps to take. we don't have to remember where the docs are.

**Review by dustymabe — APPROVED** [review]
LGTM - maybe do a test run to make sure the formatting looks good in the slack message.

Thanks for doing this.

**Comment by aaradhak** [comment]
Tested & working as expected, posting the [slack message](https://redhat-internal.slack.com/archives/C045GH4M2HY/p1725472070501289) for reference

**Pull request merged**